### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by SamArsanios**
In this project, I build a weather app that tells the user the current weather statistics of a given city. This project was built with Javascript. The API call is made from openweathermap.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+- Clone the repo typing:
+
+```
+$ git clone git@github.com:SamArsanios/weather-app.git
+```
+
+- Install dependencies:
+
+```
+$ npm install
```
Comment (descholar-ceo) [inline README.md:33]: [optional] Please clarify what to do after `npm i`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    // Load weather based on search query
+    await weatherData.loadWeather(city);
+
+    // Render weather data
+    view.renderWeather(weatherData.weather.data);
+  } catch (err) {
+    console.error(err);
+    view.showMessage(err.message);
+  }
+};
```
Comment (descholar-ceo) [inline src/index.js]: [required] Kindly remove all of the commented codes

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+const weatherLocation = function () {
+  view.showSpinner();
+
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(loadLocation, () => {
+      view.showMessage();
+    });
+  }
+};
```
Comment (descholar-ceo) [inline src/index.js]: - [x]  [required] usage of arrow functions instead of the function keyword

File: src/weatherData.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  weather.data = {
+    name: data.name,
+    country: data.sys.country,
+    temp: Math.round(data.main.temp),
+    temperature: Math.round(data.main.feels_like),
+    humidity: data.main.humidity,
+    sunrise: convertTime(data.sys.sunrise),
+    sunset: convertTime(data.sys.sunset),
+    date: convertTime(data.dt, 'full'),
+    // iconBig: `https://openweathermap.org/img/wn/${data.weather[0].icon}@4x.png`,
```
Comment (descholar-ceo) [inline src/weatherData.js]: - [x] [required] Kindly let's remove this commented codes

File: src/weatherData.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        + data.weather[0].description.slice(1),
+    unit: UNIT,
+  };
+
+  const data2 = await AJAX(`${REST_COUNTRY_URL}${weather.data.country}`);
+
+  weather.data.flag = data2[0].flag;
+  // } catch (err) {
+  // throw err;
+  // }
```
Comment (descholar-ceo) [inline src/weatherData.js]: - [x] [required] Kindly let's remove this commented codes

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @SamArsanios  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented all of the requirements so carefully, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] The weather app should change its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from Odin)
- [x] You sgould use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from Pathwright)
    - [x] usage of arrow functions instead of the function keyword
- [x] Kindly remove the commented codes
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**